### ISSUE TITLE
Browser doesn't sort numerically

### ISSUE THREAD

**Issue by LukeCEL** [CelestiaProject/Celestia#628]
Both the 1.6.1 browser and the 1.7 QT Celestial Browser treat numbers as if they should be sorted alphabetically. For example, instead of this putting these objects in this order:
- NGC 1
- NGC 2
- NGC 3
- NGC 4
- NGC 5

etc., they put objects in this order:
- NGC 1
- NGC 10
- NGC 100
- NGC 1000
- NGC 1001

**Comment by SevenSpheres** [comment]
I mean, this isn't really an issue for me. Also, why is this tagged as "qt", since it isn't Qt-specific?

**Comment by LukeCEL** [comment]
> Also, why is this tagged as "qt", since it isn't Qt-specific?

Good point

**Comment by LukeCEL** [comment]
Update: @eyvallah fixed the bug for 1.7, now it only exists in 1.6.x.

**Comment by SevenSpheres** [comment]
Why is this issue still open, since this has been done for the Qt, native Win, Android, and presumably macOS/iOS frontends? I see there's a "gtk" label, so has this not been done for the Gtk frontend?

**Comment by 375gnu** [comment]
No, gtk and native win interfaces were not updated (and i doubt they will be).

**Comment by SevenSpheres** [comment]
This is the Windows native frontend.
![nativewinsort](https://user-images.githubusercontent.com/56284207/144753969-47ce6e83-04d6-4043-8f81-2e47a6dc59cd.png)

**Comment by 375gnu** [comment]
@SevenSpheres that's not browser. That's a search console.

**Comment by SevenSpheres** [comment]
Well...
* In 1.6.x on Windows the search console sorts alphabetically, so this has been changed
* The Windows native frontend has no DSO browser, and the star browser apparently can't be sorted by name at all

**Comment by 375gnu** [comment]
Let's keep it open for a while until we decide the future of win32 & gtk frontends.

**Comment by 375gnu** [comment]
gtk was removed, win-native is doomed
